### DISCUSSION TITLE
create-next-app doesn't have the all file structure

### DISCUSSION THREAD

**Discussion by sefailyasoz95** [vercel/next.js#30229] [Help] [answered]
Hi guys, I recently decided to take a look and learn the next.js and in every tutorial I watched once they run the "npx create-next-app app-name" command they got all the file structure with js files and all. but when I do this all I got node_modules, package.json and yarn.lock files. As you can imagine I cannot run any command to start the app. I did "npm install -g create-next app". What am I missing here ?

**Comment by gaston-flores** [comment]
Those are very different commands. `npm install -g create-next app` will install the packages `create-next` and `app` globally with NPM. You want this one:

```sh
npx create-next-app
```

This will run [create-next-app](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) without installing it.

**Reply by sefailyasoz95** [reply]
I know they do. I was trying to make it clear that I installed it globally and then run the create-next-app, just like for create-react-app. 
AND STILL DIDN'T WORK.

**Comment by sefailyasoz95** [comment, marked as answer]
SOLVED. My node version was 12.x and I didn't get any folder like public and styles. There were just node_modules and package.json. I updated note to version 14.x run the " yarn create next-app " and now it is working just fine. PS: I tried yarn start after creating the project gave me an error then I run the yarn build first and then yarn start and now project is working

**Reply by gaston-flores** [reply]
Glad you figured it out!

As for `yarn start`, you can run the project in development mode with `yarn dev`. `start` is production mode, after you've run `build` like you did.

**Reply by Naruto-19** [reply]
I did this but still stuck.
